deploy/cloudwash_job.yaml

```diff
@@ -66,6 +64,16 @@ objects:
                       secretKeyRef:
                         key: location
                         name: provisioning-azure-auth-secret
+                  - name: CLEANUP_AWS__AUTH__ACCESS_KEY # AWS creds
+                    valueFrom:
+                      secretKeyRef:
+                        key: access_key_id
+                        name: cloudwash-secret
```

Comment:
Do you have an app-interface MR for the namespace to include this secret?

Reply:
yep here
https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/89921/diffs